src/main/webclient/src/components/layout/monitoring-tables/all-devices-view/all-devices-view.tsx

```diff
@@ -8,11 +8,15 @@ interface IAllDevicesView {
 }
 
 export const AllDevicesView = ({ model, inactiveOnly }: IAllDevicesView) => {
-  return (
-    <Accordion defaultIndex={[0, 1, 2, 3]} allowMultiple>
-      {model.bridges.map((bridge) => {
-        return <BridgeRowView bridge={bridge} inactiveOnly={inactiveOnly} />;
-      })}
-    </Accordion>
-  );
+  if (model.getDevicesCount()) {
+    return (
+      <Accordion defaultIndex={[0, 1, 2, 3]} allowMultiple>
```

Comment:
to do: zmienić default index - stworzone pod testowanie, nie będzie działać dla więcej niż 4 urządzeń